geopy/distance.py

```diff
@@ -2,26 +2,39 @@
 .. versionadded:: 0.93
 
 Geopy can calculate geodesic distance between two points using the
-[Vincenty distance](https://en.wikipedia.org/wiki/Vincenty's_formulae) or
-[great-circle distance](https://en.wikipedia.org/wiki/Great-circle_distance)
-formulas, with a default of Vincenty available as the function
+`geodesic distance
+<https://en.wikipedia.org/wiki/Geodesics_on_an_ellipsoid>`_ or
+`great-circle distance
+<https://en.wikipedia.org/wiki/Great-circle_distance>`_
+formulas, with a default of the geodesic distance available as the function
 `geopy.distance.distance`.
 
 Great-circle distance (:class:`.great_circle`) uses a spherical model of
 the earth, using the mean earth radius as defined by the International
-Union of Geodesy and Geophysics, (2*a + b)/3 = 6371.0087714150598
+Union of Geodesy and Geophysics, (2\ *a* + *b*)/3 = 6371.0087714150598
 kilometers approx 6371.009 km (for WGS-84), resulting in an error of up
 to about 0.5%. The radius value is stored in
 :const:`distance.EARTH_RADIUS`, so it can be customized (it should
 always be in kilometers, however).
 
-Vincenty distance (:class:`.vincenty`) uses a more accurate ellipsoidal model
-of the earth. This is the default distance formula, and is thus aliased as
-``distance.distance``. There are multiple popular ellipsoidal models, and
-which one will be the most accurate depends on where your points are located
-on the earth. The default is the WGS-84 ellipsoid, which is the most globally
-accurate. geopy includes a few other
-models in the distance.ELLIPSOIDS dictionary::
+The geodesic distance is the shortest distance on the surface of an
+ellipsoidal model of the earth.  There are two algorithms used to
```

Comment:
> There are two algorithms used to compute this distance.

Aren't there three now? Geodesic, Vincenty and GreatCircle.

GreatCircle is not even mentioned in this introduction, so maybe it makes sense to rephrase this sentence to not include a count of the algorithms.